.github/workflows/cvdremote_linux_continuous.yaml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   build-linux:
     name: cvdremote-build-linux
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
The whole point of running in ubuntu 18.04 was to ensure **that** version remained usable, is there no other way around this?

Reply:
removed these actions, users can use `go tool` to install `cvdr` rather than downloading it.